components/pwm/rgb_led/definition.json

```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "RGB LED",
+    "pwmSetting": "fixedFrequency",
+    "visualization": {
+      "type": "color-picker"
```

Comment:
@brentru I see type: `color-picker` and I'm realizing we don't have that in the discussion/diagrams on Basecamp. Want to:
- ...design out how it work for me?
- ...change it to a different type?
- ...remove the component?